.github/workflows/ci.yaml

```diff
@@ -17,6 +17,8 @@ jobs:
 
   unit-test:
     name: Unit tests
+    needs:
+      - lint
```

Comment:
SUGGESTION

What about putting unit-test and linting together in the same job but different steps? This way one could do checkout and tox installation only once

Reply:
Nice idea, but they both take less than a minute and it's easy to tell what's happening at a glance if they're separate jobs - if the setup stage was longer, I'd do this, but I don't think it's necessary